ui/src/app/kvm/views/KVMDetails/LxdProject/ProjectVMs/VMsTable/VMsTable.tsx

```diff
@@ -126,106 +124,115 @@ const VMsTable = ({ currentPage, id }: Props): JSX.Element => {
     dispatch(machineActions.setSelected(machineIDs));
   });
 
-  if (!pod || loading) {
+  if (loading) {
     return <Spinner text="Loading..." />;
   }
   return (
-    <MainTable
-      className="vms-table"
-      headers={[
-        {
-          className: "name-col",
-          content: (
-            <div className="u-flex">
-              <GroupCheckbox
-                items={machineIDs}
-                selectedItems={selectedIDs}
-                handleGroupCheckbox={handleGroupCheckbox}
-              />
-              <div>
-                <TableHeader
-                  currentSort={currentSort}
-                  data-test="name-header"
-                  onClick={() => updateSort("hostname")}
-                  sortKey="hostname"
-                >
-                  Name
-                </TableHeader>
+    <>
+      <MainTable
```

Comment:
The only thing that's changed in this large diff is that now `MainTable` is wrapped in a fragment so that the "No VMs" `Strip` can be added to the bottom